liferay-workspace/tests/src/test/java/com/liferay/blade/tests/BladeCLI.java

```diff
@@ -91,7 +91,7 @@ public static String getLatestBladeCLIJar() throws Exception {
 
 			Map<String, String> repoMap = new HashMap<>();
 			repoMap.put("name", "index1");
-			repoMap.put("locations", "http://cloud-10-50-0-245:8081/job/liferay-blade-cli/lastSuccessfulBuild/artifact/build/generated/p2/index.xml.gz");
+			repoMap.put("locations", System.getProperty("buildURL") + "/job/liferay-blade-cli/lastSuccessfulBuild/artifact/build/generated/p2/index.xml.gz");
```

Comment:
Can you call this blade url?  also in the `liferay-workspace/tests/build.gradle` can you add into the test task the systemProp 'bladeURL' and set it to the default of cloudbees and you can override that in jenkins